### DISCUSSION TITLE
milestone 3-3

### DISCUSSION THREAD

**Discussion by Keiven-Soto** [uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes#388] [General]
Sprint issues reviewed, scrum prep meetings and weekly meetings

**Comment by frances-sepulveda** [comment]
Started researching to implement the final board for the game, did a lecture topic task

**Comment by Saul-Caban-UPRM** [comment]
Weekly meeting, deciding what LTT to do

**Comment by nsfrontera** [comment]
Finished a few tasks, working on making a pull request of one of them and working on starting the rest of my tasks

**Comment by ilikecheese117** [comment]
Finished sprint issues regarding markdown documentation.

**Comment by Adriana-Isabel** [comment]
Attended the weekly meeting and finished an issue about the links that correspond to the Contact Us, Privacy Policy, FAQs. Brainstormed on what LTTs to do.

**Comment by print-Eruki** [comment]
Fix documentation issues, ltt tasks and created new tasks

**Comment by sandy-marrero** [comment]
Completed first milestone guidelines task. Working on additional documentation fixes and LTT.

**Comment by ianramospr** [comment]
Started figuring out my next issue for this milestone.

**Comment by DavidCuevas3** [comment]
Brainstormed on possible tasks to perform for the rest of the milestone.

**Comment by PinkSylvie** [comment]
After the Milestone 2 documentation grade and feedback was given I created a task for every feedback given to ensure they get worked on before the end of the milestone

**Comment by AngelCIICMorales** [comment]
Reviewed tasks and held meetings.

**Comment by FrancesC8** [comment]
Closed a documentation fix for Synopsis. Will continue work on remaining tasks for milestone.

**Comment by AnthonyMed13** [comment]
Worked on documentation task

**Comment by JadiellMV** [comment]
This week I started working on my LTTs

**Comment by G4vp** [comment]
Worked on  lecture topic tasks

**Comment by ebrain-mendoza121** [comment]
Worked on tasks related to handling multiplayer connection

**Comment by Negroni1** [comment]
Worked on a diagram that demonstrates how multiplayer domino implements the aggregates design pattern

**Comment by fernandopizarro1** [comment]
Reviewed tasks and helped in the creation of new tasks and ideas for my team

**Comment by Gaby-Concepcion** [comment]
Attended weekly meeting and finished a LTT while beginning preparations for the milestone 3 video presentation.

**Comment by RicardoJRSs** [comment]
Worked on lecture topic tasks

**Comment by Joaquin939** [comment]
This week i worked on finding issues for this sprint

**Comment by Jvstinoo** [comment]
Worked on lecture topic tasks.
